### ISSUE TITLE
Adding a feed from a URL

### ISSUE THREAD

**Issue by nhuillard** [FredJul/Flym#398]
I didn't add feeds very often, but always from plain URLs, and it worked at least on older 1.x.
Now, pasting an URL in the "add feed" dialog doesn't do anything (it only seems to filter the pre-built list). Sharing an URL to Flym from a browser doesn't either.
The older behaviour was 1) get the title of the feed from the <title> of the page or RSS, 2) identify the feeds provided in the website's headers 3) probably allow to select one feed if many are provided.
Firefox mobile also allows to subscribe to feeds, but only with online services. Maybe Flym could provide it's offline service too, if possible.
Many thanks for this great software, that I use since a very long time!

**Comment by omehegan** [comment]
Yes, I can't figure out how to add blogs that don't display an RSS link.

**Comment by keunes** [comment]
From the just closed #468: one site where the feed is not proposed/recognised is http://opensource.com
And from #606: https://www.units.it/feed

**Comment by papernoise** [comment]
Just chiming in to say that I have the same issue and would love to be able to just copy-paste the feed url to add it to the app.
I've been trying out all the feed readers on f-droid and this one is definitely the one that works best!

**Comment by levirs565** [comment]
You can generate OPML from URL. You can use https://opml-gen.ovh/ to generate OPML from URL list. Then click Download to Download OPML. Then you can import OPML from Flym
